packages/Tag/stories/Tag.stories.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable react/no-array-index-key */
+import React from "react";
+import CalendarIcon from "@paprika/icon/lib/Calendar";
+import ExclamationCircle from "@paprika/icon/lib/ExclamationCircle";
+import tokens from "@paprika/tokens";
+import Avatar from "../../Avatar/src";
+
+import Tag, { Tags } from "../src";
+import * as types from "../src/types";
+
+export default {
+  title: "Tag",
+  component: Tag,
+};
+
+export function TagsExample() {
```

Comment:
Can you extract this to `/stories/Examples/Variations.js` and use the `Tag.stories.js` as an index/manifest for stories in the root folder (of the Storybook navigation)?

No need to go crazy with this PR, I think the Variations story has the most value of anything, so you can probably even drop the `Basic` one. 

I would recommend adding this component to the `storyTree` though, so it fits into the Storybook IA.

This guide is still pretty rough, but it can probably still help with the details:
https://github.com/acl-services/paprika/wiki/Storybook-Curation-Guide